_dev/src/ts/appUI/pages/UpdatePagePostUpdate.ts

```diff
@@ -17,11 +17,43 @@
  * @license   https://opensource.org/licenses/AFL-3.0 Academic Free License version 3.0
  */
 import StepPage from './StepPage';
+import api from '../api/RequestHandler';
 
 export default class UpdatePagePostUpdate extends StepPage {
   protected stepCode = 'post-update';
 
   public mount() {
     this.initStepper();
+    this.#handleHydrate();
+  }
+
+  #handleHydrate = () => {
+    this.#addListenerToDialogConfirmModuleManagerLink();
+  };
+
+  #onClickDialogLink = async (event: MouseEvent) => {
+    const target = event.target as HTMLAnchorElement;
+
+    // Checks if the clicked element is an <a> tag pointing towards an ID
+    if (!target || target.tagName !== 'A' || !target.hash) {
+      return;
+    }
+
+    event.preventDefault();
+
+    const hashRoute = target.hash.substring(1);
+    await api.post(hashRoute);
+  };
+
+  #addListenerToDialogConfirmModuleManagerLink = () => {
+    this.#dialogConfirmModuleManagerLink.addEventListener('click', this.#onClickDialogLink);
```

Comment:
I usually find a equivalent `removeEventListener` called by the beforeDestroy event